client/checkout/blocks/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerPaymentMethod } from '@woocommerce/blocks-registry';
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME } from '../constants.js';
+import { getConfig } from './../utils.js';
+import WCPayAPI from './../api';
+import WCPayFields from './fields.js';
+
+// Create an API object, which will be used throughout the checkout.
+const api = new WCPayAPI( {
+	publishableKey: getConfig( 'publishableKey' ),
+	accountId: getConfig( 'accountId' ),
+} );
+
+// Add the payment method to the blocks registry.
+registerPaymentMethod(
```

Comment:
Should there be a version check around this? E.g. old version of WooCommerce/blocks that doesn't have blocks payment methods API. (Maybe there's already a higher-level version check in this plugin somewhere?)

Reply:
I don't think this file will get loaded in the client if it's not registered on the server and it won't get registered on the server if the action server side is not available. So I think they should be good here.